core/src/main/java/feign/Response.java

```diff
@@ -44,35 +46,150 @@
   private final String reason;
   private final Map<String, Collection<String>> headers;
   private final Body body;
+  private final Request request;
 
+  @Deprecated
```

Comment:
Old private constructor-- kept it around to have backwards compatibility with the factory methods but added a line to set the Request field to null